### PR TITLE
awesome-books es6 modules

### PR DESCRIPTION
**Pull request by Khaled-AlKhateeb**
## Tasks Completed ✔️ : 

- Created a module directory,
- Created all modules using JavaScript ES6.
- made sure the export and import work properly on each js file.
- fix linter errors.

### CODE REVIEW

File: modules/add-new-book.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  bookEntries.author = authorInput.value;
+
+  library.add(bookEntries);
+
+  titleInput.value = '';
+  authorInput.value = '';
+
+  window.localStorage.setItem('books', JSON.stringify(library.booksArr));
+  drawOnScreen(bookEntries);
+}
```
Comment (mmsesay) [inline modules/add-new-book.js]: - [x] This is very handy in getting the form input values. However, I will suggest you refactor it and utilize ES6 arrow function syntax for[ readability and more](https://www.freecodecamp.org/news/when-and-why-you-should-use-es6-arrow-functions-and-when-you-shouldnt-3d851d7f0b26/).

Example: 
```js
const getInputValues = () => {
 // add contents here
}
```

File: modules/drow-on-screen.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  removeBtn.classList.add('remove-btn');
+  removeBtn.innerHTML = 'Remove';
+  removeBtn.setAttribute('onclick', 'removeBook(this.id)');
+  bookTitle.innerHTML = '"' + obj.title + '" by ' + obj.author;
+  bookDiv.setAttribute('id', idString);
+  removeBtn.setAttribute('id', rmvBtn);
+  bookDiv.appendChild(bookTitle);
+  bookDiv.appendChild(removeBtn);
+  addBook.appendChild(bookDiv);
+}
```
Comment (mmsesay) [inline modules/drow-on-screen.js]: - [x] Please change this function an ES6 arrow function syntax.

File: modules/localstorage-onload.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+  window.addEventListener('load', () => {
+    const localStorageItem = window.localStorage.getItem('books');
+    if (localStorageItem) {
+      library.booksArr = JSON.parse(localStorageItem);
+      library.booksArr.forEach((element) => {
+        drawOnScreen(element);
+      });
+    }
+  });
+}
```
Comment (mmsesay) [inline modules/localstorage-onload.js]: - [x] Good job working on this function 👍. As I suggested above please change it to an arrow function syntax.

File: modules/select-tab.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+function selectAdd() {
+  listTab.style.display = 'none';
+  addNewTab.style.display = 'block';
+  contactTab.style.display = 'none';
+}
+function selectContact() {
+  listTab.style.display = 'none';
+  addNewTab.style.display = 'none';
+  contactTab.style.display = 'flex';
+}
```
Comment (mmsesay) [inline modules/select-tab.js]: - [x] Please update these functions also to arrow syntax.

**Review by mmsesay — CHANGES_REQUESTED** [review]
Hi @Khaled-AlKhateeb, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
 - Good Readme :heavy_check_mark: 
 - No linter error :heavy_check_mark: 
 - Good PR title :heavy_check_mark: 
 - Good PR summary :heavy_check_mark:  


## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mikethreels — APPROVED** [review]
Hi Khery, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- good job following the instructions of the previous CR
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**